CHANGES/4102.misc

```diff
@@ -0,0 +1 @@
+web.Application object now has a boolean value of True
```

Comment:
web.BaseRequest is also affected, isn't it?

Reply:
oops :smile: